client/src/components/modals_and_popovers/FixedPopover.stories.js

```diff
@@ -0,0 +1,37 @@
+import React from "react";
+
+import { FixedPopover } from "./FixedPopover.js";
+
+export default {
+  title: "FixedPopover",
+  component: FixedPopover,
+};
+
+const Template = (args) => <FixedPopover {...args} />;
+
+export const Basic = Template.bind({});
+Basic.args = {
+  show: true,
+
+  // text
+  title: "Title",
+  body: "Body",
+  header: "Header",
+  subtitle: "Subtitle",
+  footer: "Footer",
+
+  // booleans
+  restore_focus: false,
+  close_button_in_header: false,
+  close_text: false,
+
+  // dimensions
+  max_body_height: "3em",
+
+  // css
+  dialog_position: "",
+  additional_dialog_position: "",
+
+  // clock
+  auto_close_time: 1000,
```

Comment:
The use case with `auto_close_time` is pretty different from the standard use. Should be a separate story. Can use the `on_close_callback` to log when the timer runs out. Not sure why the timer display isn't appearing in the current story either, something to look at when making the timed version's story.

Reply:
Done!